packages/typespec-azure-resource-manager/lib/common-types/types.tsp

```diff
@@ -218,9 +218,8 @@ model OperationStatusResult {
 
   /** Fully qualified ID of the resource against which the original async operation was started. */
   @visibility(Lifecycle.Read)
-  @format("arm-id")
   @added(Versions.v5)
-  resourceId?: string;
```

Comment:
My question is how this impacts our common types output in typespec-autorest.  Adding @AlitzelMendez  as a reviewer for comment

Reply:
It changes nothing, we do common type replacement.
`test: Azure.ResourceManager.CommonTypes.OperationStatusResult;` produces 
```
"test": {
          "$ref": "../../../common-types/resource-management/v5/types.json#/definitions/OperationStatusResult"
        }
```

Reply:
thanks Pan!! I will just double check and if everything looks good I will approve